src/index.js

```diff
@@ -10,5 +20,162 @@ module.exports = {
   extractImports:build.extractImports,
   createImportBlock:build.createImportBlock,
   sortFiles:build.sortFiles,
-  cleanGeneratedCode: build.cleanGeneratedCode
+  cleanGeneratedCode: build.cleanGeneratedCode,
+  docShapeDefs: docShapeDefs.shapeDefs,
+  docShape: docShape.shape
 };
+
+const argv = require('yargs')
+  .command('ts-build-all', 'Build multiple versions of the project', {
+      project: {
+        alias: 'p',
+        describe: 'TypeScript project file or folder',
+        type: 'string',
+        default: 'tsconfig.build.json'
+      },
+      outDir: {
+        alias: 'out',
+        describe: 'Output directory for compilations',
+        type: 'string',
+        default: 'dist'
+      },
+      'continue-when-failed': {
+        describe: 'Do not bail when one compilation fails',
+        type: 'boolean',
+        default: false
+      },
+      'clean-before': {
+        describe: 'Clean outdir before compiling',
+        type: 'boolean',
+        default: false
+      }
+    },
+    function(argv) {
+      if (argv.cleanBefore) {
+        rimraf(argv.outDir);
+      }
+
+      const variations = [
+        { module: "amd" },
+        { module: "commonjs" },
+        { module: "es2015", directory: "native-modules" },
```

Comment:
ah- I see the distinction now: `tsc --outDir dist/native-modules --module es2015` vs `tsc --outDir dist/es2015 --target es2015 --module es2015`.  I think I broke this in the ux build 😬